### ISSUE TITLE
After creating a new playlist, keyboard focus should be on the new playlist's text

### ISSUE THREAD

**Issue by Pointedstick** [gnumdk/lollypop#1093]
### Environment
• Lollypop version: 0.9.233
• Gtk Version: 3.22.11-1.1
• Linux distribution: openSUSE Tumbleweed
• Desktop environment: KDE Plasma 5.9.5

### Bug
Steps to reproduce:
1. Click "Playlists"
2. Click "New Playlist"

Expected results:
The new playlist's text should be selected so you can immediately start typing to name it.

Actual results:
The new playlist is named "New playlist 1" and keyboard focus is moved to the playlist below it. Two unnecessary clicks are required to highlight the new playlist's text so that you can name it.
